test152/test152.ino

```diff
@@ -24,7 +24,7 @@ void setup(){
 
     LCD_Init();
     PWM_Init();
-    LCD_ShowPICALL(pic_XVF);
+    LCD_ShowPICALL(xvf152.bmp);
```

Comment:
This is an error in "LCD_ShowPICALL(pic_XVF)" - a picture's name is "xvf152.bmp".

Reply:
It can't access the bmp files directly, sadly. It was correct as pic_XVF - there was an array called pic_XVF that got clobbered on your local branch somehow during PR #32 which is why it looks broken. You can see the array got accidentally removed at the bottom of the file in the diff in PR 32.
I'm not entirely sure how that happened! You might be able to figure it out and fix it, and if not I can just re-add the array.